vizro-core/tests/tests_utils/e2e_asserts.py

```diff
@@ -0,0 +1,56 @@
+import shutil
+from pathlib import Path
+
+import cv2
+import imutils
+from hamcrest import assert_that, equal_to
+
+
+def _compare_images(expected_image, result_image):
+    """Comparison process."""
+    difference = cv2.subtract(expected_image, result_image)
+    blue, green, red = cv2.split(difference)
+    assert_that(cv2.countNonZero(blue), equal_to(0), reason="Blue channel is different")
+    assert_that(cv2.countNonZero(green), equal_to(0), reason="Green channel is different")
+    assert_that(cv2.countNonZero(red), equal_to(0), reason="Red channel is different")
+
+
+def _create_image_difference(expected_image, result_image):
+    """Creates new image with diff of images comparison."""
+    diff = expected_image.copy()
+    cv2.absdiff(expected_image, result_image, diff)
+    gray = cv2.cvtColor(diff, cv2.COLOR_BGR2GRAY)
+    for i in range(0, 3):
+        dilated = cv2.dilate(gray.copy(), None, iterations=i + 1)
+    (t_var, thresh) = cv2.threshold(dilated, 3, 255, cv2.THRESH_BINARY)
+    cnts = cv2.findContours(thresh, cv2.RETR_LIST, cv2.CHAIN_APPROX_SIMPLE)
+    cnts = imutils.grab_contours(cnts)
+    for contour in cnts:
+        (x, y, width, height) = cv2.boundingRect(contour)
+        cv2.rectangle(result_image, (x, y), (x + width, y + height), (0, 255, 0), 2)
+    return result_image
+
+
+def make_screenshot_and_paths(browserdriver, request):
```

Comment:
```suggestion
def make_screenshot_and_paths(browserdriver, request_node_name):
```
and then just use `request_node_name` directly here rather than passing in the whole `request` object.